Week-5/Solution6.js

```diff
@@ -0,0 +1,25 @@
+const defer = (res, time) =>
+  new Promise(function (resolve) {
+    setTimeout(function () {
+      resolve(res);
+    }, time);
+  });
+
+const returnMyPromise = async () => await defer("done 1", 1000);
+const returnMyPromise2 = async () => await defer("done 2", 2000);
+const returnMyPromise3 = async () => await defer("done 3", 3000);
+
+// Promise.all([returnMyPromise(), returnMyPromise2(), returnMyPromise3()]).then(
+//   (allres) => {
+//     console.log(allres);
+//   }
+// );
+
+async function doTasks() {
```

Comment:
This function will not give us the indented output as per the assignment question. Can you please modify it to include the execution of all 3 functions? Can you also make sure it executes for both `promises` and `generators`?